src/lib/y2packager/clients/inst_product_license.rb

```diff
@@ -32,8 +33,36 @@ def main
       #
       # @return [Y2Packager::Product]
       # @see Y2Packager::Product.selected_base
-      def selected_product
-        @selected_product ||= Y2Packager::Product.selected_base
+      def product
+        @product ||= Y2Packager::Product.selected_base
+      end
+
+      # Determines whether the license should be shown
+      #
+      # @return [Boolean]
+      def show_license?
+        multi_product_media? && available_license?
+      end
+
+      # Determines whether a multi-product media is being used
+      #
+      # This client only makes sense when using a multi-product media.
```

Comment:
Not sure about the meaning of that sentence

Reply:
It means that this client is only used in multiproduct installation medias like LeanOS + SLES + SLED. For single product, the license will be shown in the complex welcome screen.

But we could make it easier and drop this condition. After all, the client won't be on control file of single product medias.